CONTRIBUTING.md

```diff
@@ -33,7 +39,13 @@ Follow these steps to build a new version of the plugin:
 
 ### Environment setup
 
-Make sure you can build the plugin by following the steps above.
+First, make sure you can build the plugin by following the steps above.
+
+Then, make sure you have installed ```Gulp```.
```

Comment:
LOL I saw the triple ticks for some inline code snippet somewhere and started using it everywhere...time to fix!